src/App.tsx

```diff
@@ -131,20 +145,6 @@ export function App() {
           />
         </Box>
 
-        <Box>
-          <Label htmlFor="chat-client">Chat Client</Label>
-          <Select
-            id="chat-client"
-            value={chatClient}
-            onChange={(event) =>
-              setChatClient(event.target.value as ChatClient)
-            }>
-            <option value="discord">Discord</option>
-            <option value="telegram">Telegram</option>
-            <option value="browser">Browser</option>
-          </Select>
-        </Box>
-
```

Comment:
fascism

Reply:
?

Reply:
<img width="558" alt="Screen Shot 2021-09-09 at 2 13 27 AM" src="https://user-images.githubusercontent.com/6154901/132601929-5e58e272-dfaa-4dde-843c-a9430aaed119.png">

Reply:
Sorry, I added back the client select